processing/data_collection/gazette/spiders/rn_macaiba.py

```diff
@@ -0,0 +1,76 @@
+import re
+
+import dateparser
+import datetime as dt
+
+import scrapy
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class RnMacaiba(BaseGazetteSpider):
+    start_date: dt.date
```

Comment:
I think this one should be set by the base class, as it can be a command line argument

Reply:
Yes, that's right.

Reply:
Done.